Coalition: Alpha encounter (#5904)
endless-sky/endless-sky: data/coalition/coalition missions.txt

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

mission "Coalition: Alpha Encounter"
	minor
	invisible
	to offer
		has "Coalition: First Contact: done"
		has "remembrance day: offered"

> 		random = 1

Reviewer: 1) If you want to check equivalence, use `==`, not `=`, which is assignment.
2) You can still just use `random < 1` for a 1% chance.
```suggestion
		random < 1
```

	source
		government "Coalition"
		not attributes "station"
		not planet "Secret Sky" "Remote Blue" "Bloptab's Furnace" "Ahr"
	on offer
		conversation

> 			`You walk around the spaceport for a few minutes, when suddenly you're tackle into the ground, with whoever attacked you pinning you down, holding your arms behind your back to restrain you. When you manage to look back, you see a man of human appearance, with very long hair and wearing an intricate necklace of gold. His most notable trait, however, is his green skin - an Alpha.`

Reviewer 2: ```suggestion
			`You walk around the spaceport for a few minutes, when suddenly you're tackled into the ground, with whoever attacked you pinning you down, holding your arms behind your back to restrain you. When you manage to look back, you see a man of human appearance, with very long hair and wearing an intricate necklace of gold. His most notable trait, however, is his green skin - an Alpha.`
```

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

mission "Coalition: Alpha Encounter"
	minor
	invisible
	to offer
		has "Coalition: First Contact: done"
		has "remembrance day: offered"
		random < 1
	source
		government "Coalition"
		not attributes "station"
		not planet "Secret Sky" "Remote Blue" "Bloptab's Furnace" "Ahr"
	on offer
		log "Minor People" "Alitis" `Alitis, formerly known as Athis, is an Alpha who lives in the Coalition. Centuries ago, he stole a jump drive from an alien ship and used it to explore, but after some sort of encounter in the galactic east, he decided to give up on his former life.`
		conversation

> 			`You walk around the spaceport for a few minutes, when suddenly you're tackled into the ground, with whoever attacked you pinning you down, holding your arms behind your back to restrain you. When you manage to look back, you see a man of human appearance, with very long hair and wearing an intricate necklace of gold. His most notable trait, however, is his green skin - an Alpha.`

Reviewer 3: This sequence of events could use work, and being broken into more sentences.

Author: I did some edits to break it up a little more, but I imagine it's not fully what you meant.
Which parts exactly need more work, or be broken up into more sentences?

> 			`You walk around the spaceport for a few minutes, when suddenly you're tackled into the ground. You try to get up, but whoever attacked you pins you down, holding your arms behind your back to restrain you. When you manage to look back, you see a man of human appearance, with very long hair and wearing an intricate necklace of gold. His most notable trait, however, is his green skin - an Alpha.`

Author: That second option works best I'd say.

> 			`You walk around the spaceport for a few minutes, when suddenly you're tackled into the ground. You try to get up, but whoever attacked you pins you down, holding your arms behind your back to restrain you. When you manage to look back, you see a man of human appearance, with very long hair and wearing an intricate necklace of gold. His most notable trait, however, is his green skin - an Alpha.`

Reviewer 4: Updated.

> 			`	Your eyes don't meet for long, though, as he looks to his side and lets go of your arms to try and block an Arach that swoops in with a tackle, knocking him off of you. They wrestle for a while, as you get up, but it doesn't take long for the Arach Heliarch to restrain the Alpha, with all eight legs firmly holding his arms to his torso, as the Arach rests her entire body weight on the Alpha's back. A Saryd Heliarch approaches you, telling you that "Keep your distance, you must," and then looking at the Alpha. "What reason for attacking this person, did you have?"`

Reviewer 3: This Arach would be easily toppled, any more than 6 legs is asking to end up on the floor.

Author: Eh, they're heavy and wide enough I doubt it'd be an easy task.

Reviewer 3: If they're getting all 8 legs onto a single person's torso, then they've created a fulcrum. It doesn't matter how heavy they are, in fact the heavier they are the worse off they are. It would be trivial to topple them. They require at least two other points of contact with something that is not their target in order to safely and effectively restrain them.

Author: They've got long legs, a lot of joints would be touching the ground.

Reviewer 3: They can't apply their weight through those legs, if they're touching the ground some other way with a different joint. That's not how this works.

> 			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question and looks at you instead. He asks, "You're with the Navy, aren't you? Finally figured out I was here?"`

Author: For that first paragraph I'd drop the bit about the quicksteps, I'd expect an Alpha to be more capable at catching someone by surprise, and quick footsteps in a spaceport marketplace shouldn't be a source of concern in the Coalition, where one expects to be safe, and wouldn't assume any quick footsteps would mean a potential attacker.

Most of my problems are with the suggested second paragraph.

- I always forget this so correct me if I've got this wrong again, but aren't all Alphas male? If so then `The Alpha is a man` is redundant.
- I feel it's more a testament to the Arach's abilities if we don't explicity say "even though Alphas are so strong the Arach can restrain this one pretty easily."
- I don't see the need to change the hold the Arach has on the Alpha, and this one seems like a worse version of the "six legs on him, two on the ground" that you proposed in the other comment. How would she restrain each of his legs with one of her own, wouldn't her abdomen get in the way or make that an awkward hold?

The `snarling expression` part doesn't seem necessary to me.

That entire last paragraph, no. The necklace can be mentioned earlier on, the rest of the paragraph feels almost like a fourth wall break, "oh wow look at this, this Alpha doesn't look like a big meanie psycho, weird huh?"

> 			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question and looks at you instead. He asks, "You're with the Navy, aren't you? Finally figured out I was here?"`

Reviewer 3: Re 1) So, the Alpha still catches you by surprise, it's too late to do anything about it by the time you hear the last two steps approaching you...

Re 2)
- Alphas can be female, it is simply that female alphas cannot produce viable alpha offspring.
- I know we spend a non-trivial amount of time on the discord talking about alphas and their abilities, but it is worth noting that actually in-game the player is not frequently encountering references to them. Reminding the player occasionally that they're meant to be scary af doesn't hurt, especially when you're then explicitly overpowering them.
- The first hold you had is literally impossible to maintain, so it needs to change. Precisely how you imagine this hold to go down will vary by your mental image of how fat or big the Arach is, but it is worth noting that most spiders can lift their abdomen up to get it clear of the ground when they're doing fancy footwork.

Do you want your alpha to have emotion or not? Pick a side. At present you have dispassionate coldness here and then emotion later on. 

The necklace absolutely cannot be mentioned earlier on. Why in the name of all that is holy are you noticing a necklace while tackled to the ground and struggling for your life? You've got 2-dozen things to think about before you get to that point, and including it where you have detracts from that flow. Your first opportunity to get a good look at him is after he's been pinned and you've stood up. Where else would you include any description that points to him not being entirely what you expect?

> 			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question and looks at you instead. He asks, "You're with the Navy, aren't you? Finally figured out I was here?"`

Author: Again, why would I be alarmed by some footsteps when I'm in a very safe location?

`I know we spend a non-trivial amount of time on the discord talking about alphas and their abilities, but it is worth noting that actually in-game the player is not frequently encountering references to them. Reminding the player occasionally that they're meant to be scary af doesn't hurt, especially when you're then explicitly overpowering them.`
I dunno, seems kind of out of place. If we're doing it I'd like if it wasn't said so explicitly to the player.

`The first hold you had is literally impossible to maintain`
I really doubt that.

`Do you want your alpha to have emotion or not?`
Having emotion doesn't mean just one state of being, he's not going to be a robot but he's not going to be bawling his eyes out either, that's exactly why the start of the conversation will be changed, as in that he's exceedingly casual.

Necklace can be mentioned once the Arach restrains the Alpha, or when he perks up his head to ask you the question.

> 			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question and looks at you instead. He asks, "You're with the Navy, aren't you? Finally figured out I was here?"`

Reviewer 3: `why would I be alarmed` It doesn't say that you're alarmed...

`I'd like if it wasn't said so explicitly to the player` that's probably fair, it can possibly be rephrased

`I really doubt that.` I can draw you a diagram if you'd like.

`as in that he's exceedingly casual` then it's simple enough to take off the snarling face and just swap out for an intense look

> 			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question and looks at you instead. He asks, "You're with the Navy, aren't you? Finally figured out I was here?"`

Author: What would you be reacting to if you weren't alarmed? Does first last just turn to meet whoever is passing by in a rush?

I don't need a diagram to know the Alpha isn't getting out of the hold of a giant spider holding his arms to his torso while her entire body weight rests on him.

I'd just go without a remark, having him just looking at you, but that would be the better of the two if I were to pick a remark to go with.

> 			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question and looks at you instead. He asks, "You're with the Navy, aren't you? Finally figured out I was here?"`

Reviewer 3: `Does first last just turn to meet whoever is passing by in a rush?` Probably not, but it's a natural human response to at least glance at someone if they suddenly step very near to you. That in itself is a reaction, if only a small one.

Adjusted suggestion from this convo:

```suggestion
			`You're window-browsing a shop in the spaceport when you catch the scuff of two quick footsteps coming towards you. Before you can glance in that direction though you're suddenly tackled into the ground by someone astonishingly human-shaped. You try to get up, but whoever attacked you pins you down, pressing your face into the ground and holding your arms behind your back to restrain you. As you struggle against your attacker you manage to catch sight of long hair and then, with a flash of concern, a glimpse of bright green skin - an Alpha.`
			`	You are not pressed for long though, you feel their attention shift to the side you cannot see, as an Arach swoops in with a tackle. They lets go of your arms in an attempt to block the attack, but it does not prevent the Arach from bowling them clear off of you where the two begin to wrestle in your line of sight. The Alpha is a man and, surprisingly to you, despite his strength it doesn't take long for the Arach Heliarch to restrain your attacker. The Arach firmly pins his arms to his torso with four legs, traps his legs with two, and uses her last pair to lever herself until she rests her entire body weight into pinning the Alpha.`
			`	A Saryd Heliarch approaches and tells you to keep your distance before interposing himself partially between you and your assailant. He then looks at the Alpha and asks through his translator box, "What reason for attacking this person, did you have?"`
			`	The Alpha is still struggling to try and get out of the Arach's grip, and you notice an intricate gold necklace which has splayed across his shoulder and onto the floor in the scuffle, jangling in the quiet every time he struggles as now everyone else has cleared out of the area and is giving the scene a wide berth. As you get to your feet he ignores the Saryd's question and looks at you intensely. "You're with the Navy, aren't you? Finally figured out I was here, thought you'd finish the job?"`
```

					goto notnavy
				`	"Is that why you attacked me?"`
					goto detained
			label navy
			choice
				`	"I am with the Navy, but I didn't come here looking for you."`
				`	"Is that why you attacked me?"`
					goto detained
			`	"And I'm supposed to believe that crap?" He lashes out, trying to knock the Arach off of him, but the agent only tightens the grip, and opens her mandibles, getting them close to the Alpha's neck. "Alright, alright, I get it!" The Alpha shouts, stopping his attempts to break free.`
				goto detained
			label notnavy
			`	He stares at you for a moment, as if unsure if you're telling the truth. "What the hell are you doing here then? Did you just stumble onto this place?" He stops struggling, but the Arach agent doesn't loosen her grip.`
			label detained
			`	The Saryd walks closer to the Alpha, stomping his hoofs on the ground with each step. "Whatever your motivation was, a transgression, you have committed. Punished accordingly, you shall be." He says. "Kindly follow us to the detention facility, would you, Captain?" He asks you, but doesn't wait for an answer, as more agents arrive to escort you and the Alpha to the "detention facility."`

> 			`	It's a short walk, and you end up in a building similar to a police station. The Alpha is brought before a Kimek that you guess is in charge of the facility. He asks you to give a testimony of what happened, with the Heliarch agents confirming what you say. "Your first offense in a long time this is, Alitis," the Kimek says, after checking a computer for the Alpha's data. "Three years in seclusion, your sentence is."`

Reviewer 3: Is this intentionally intended to be authoritarian and harsh? Also, is punishment time species dependent? This seems like it would be an appreciable chunk of a Kimek's lifespan.

Author: Yes. No. The chunk of time I picked didn't have to do with species or anything, I just wanted a long time for the comparatively minor offense, if three years is too little for an Alpha could increase it.

Reviewer 3: There's no point increasing it if the answer to the second question is "No" - that just makes minor infractions nearly equivalent to capital punishment for Kimek.

			`	After the short verdict, the agents bring the Alpha to a lower level of the building, with several empty cells, kept shut by thick, tough walls of some transparent material, with a few holes for airflow. They lock him in one of the cells, making him the only prisoner at the moment. They then ask that you wait a few minutes for a medic to arrive, to take care of your bruises.`

> 			`	"Don't know why they bother with that, it's not like I hit you too hard," the Alpha says. "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really, I just got a bit jumpy when I spotted you."`

Reviewer 3: If the agents took him away, why are you down next to him now? Is the medical centre in the same hallway as the cell block?

Author: Because I needed some way to get the talk started.
If having the medic's place be down by the cells would help, can do that.

Reviewer 5: Yeah, that would make it make more sense.

Author: I've added mention of it at the end of the mission.

> 			`	After the short verdict, the agents bring the Alpha to a lower level of the building, with several empty cells kept shut by thick, tough walls of some transparent material. Only a few small holes are available for airflow. They lock him in one of the cells, making him the only prisoner in the largely empty building. The agents then ask that you wait a few minutes for a medic to arrive, so that they can take care of your bruises.`
> 			`	"Don't know why they bother with that. It's not like I hit you too hard," the Alpha says. "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really. I just got a bit jumpy when I spotted you."`

Reviewer 3: Figured while I'm here I should have a go at this.
```suggestion
			`	After the short verdict which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building, with several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though all of the cells appear to be empty, and they simply lock him in the second one where accessing his cell would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive, so that they can take care of your bruises.`
			`	Cut into the transparent material are a series of small holes at head-height with a cover that can be slid across them, presumably to silence a prisoner if needed. By default they appear to be left open, and the Heliarch agents have not closed the one on his cell.`
			`	In the intervening quiet while you wait he looks at you very directly through the cell wall and says, "Don't know why they bother with that. It's not like I hit you very hard." When you don't immediately respond though his eyes drop slightly from meeting yours and he continues, "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really. I just got a bit jumpy when I spotted you."`
```

Author: I wanna make the start of the conversation less of a casual banter, to help with Wizrad's point, so I wouldn't go with that last paragraph, it is made even more emotional with the "eyes dropping" part I'd say.

Reviewer 3: Ok, alternative suggestion considering that.
```suggestion
			`	After the short verdict which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building, with several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though all of the cells appear to be empty, and they simply lock him in the second one where accessing his cell would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive, so that they can take care of your bruises.`
			`	Cut into the transparent material are a series of small holes at head-height with a cover that can be slid across them, presumably to silence a prisoner if needed. By default they appear to be left open, and the Heliarch agents have not closed the one on his cell.`
			`	In the intervening quiet while you wait he looks at you very directly through the cell wall and says, "Don't know why they bother with that. It's not like I hit you very hard." His gaze does not waver and when you don't immediately respond he continues, "Perhaps you should receive an apology. I've given up on fighting humanity left and right. You should understand that when I spotted you it was natural to perceive you as a threat."`
```

Author: The idea I had was going to trim his dialogue, something like: `While waiting, you look back at the Alpha, to find his eyes fixated on you, not looking away or even blinking. After you stare him down for a couple of seconds, he speaks up, "What is it, are you expecting some sort of apology?"`
Having him seem disinterested and starting off the conversation with a more neutral or annoyed tone.

Reviewer 3: You could go that way, though to me that seems rather hostile and lacking a bridge to the next bit of the conversation you have.

Author: Yes it's cut rather abruptly.

@W1zrad which would you say works best for making the start of the conversation with Alitis a bit more in tone with what you had in mind? If you have a suggestion of your own that would work better, could also go with that.

Reviewer 4: Hmm. I do like Arachi's abrasive tone better, but it is hard to transition to the conversation following it. What if we start with that and then just add some extra dialogue to smooth out that transition?
```suggestion
			`	After the short verdict, which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building, with several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though, all of the cells appear to be empty, and they simply lock him in the second one where accessing his cell would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive so that they can take care of your bruises.`
			`	Cut into the transparent material are a series of small holes at head-height with a cover that can be slid across them, presumably to silence a prisoner if needed. By default, they appear to be left open, and the Heliarch agents have not closed the one on his cell.`
			`	While waiting, you look back at the Alpha to find his eyes fixated on you, not looking away or even blinking. After you stare him down for a couple of seconds, he speaks up. "What is it, are you expecting some sort of apology? I'd consider my caution to be entirely justified, so you won't get one." After a long pause, he continues, "I'm sure you have questions though. I've given up on fighting humanity left and right, and it seems that I won't have anything better to do for some time. Ask away."`
```
I think this could still be improved, but here's what I'm roughly thinking. I've also made a couple comma edits on the writing where necessary.

> 			`	"Don't know why they bother with that, it's not like I hit you too hard," the Alpha says. "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really, I just got a bit jumpy when I spotted you."`

Reviewer 4: ```suggestion
			`	"Don't know why they bother with that. It's not like I hit you too hard," the Alpha says. "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really. I just got a bit jumpy when I spotted you."`
```
This is a place where we could bend rules because it's dialogue. If you'd rather keep these as commas, that's cool.

			choice
				`	"Who are you, exactly? How did you even get here?"`
				`	"What are you doing here in the Coalition?"`

> 			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s the crew I was on found some aliens somewhere far north of Earth, but we didn't get to get much from them as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting, as the battle wouldn't be favorable to us, but we managed to loot a jump drive from one of the aliens. After that we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and eventually got here."`

Reviewer 5: ```suggestion
			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s the crew I was on found some aliens somewhere far north of Earth. We didn't get to get much from them though, as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting, as the battle wouldn't be favorable to us, but we managed to loot a jump drive from one of the aliens. After that we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and after some time got here."`
```

> 			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s, the crew I was on found some aliens somewhere far north of Earth. We didn't get much from them though, as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting because the battle wasn't favorable to us, but we managed to loot a jump drive from one of the aliens. After that, we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and after some time got here."`

Reviewer 6: I feel like the dialogue here is a bit ambiguous. I'm guessing that the two groups of aliens mentioned here are the Hai and Korath respectively (at least I think so), but the statement `but we managed to loot a jump drive from one of the aliens` doesn't specify which group. I realise that you stated that only the second group had jump drives, but it'd be clearer to write `but we managed to loot a jump drive from the second group of aliens`.

> 			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s the crew I was on found some aliens somewhere far north of Earth. We didn't get to get much from them though, as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting, as the battle wouldn't be favorable to us, but we managed to loot a jump drive from one of the aliens. After that we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and after some time got here."`

Reviewer 4: ```suggestion
			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s, the crew I was on found some aliens somewhere far north of Earth. We didn't get much from them though, as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting because the battle wasn't favorable to us, but we managed to loot a jump drive from one of the aliens. After that, we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and after some time got here."`
```

			choice
				`	"You 'knew better?' What do you mean?"`
				`	"What, did you have a change of heart once you got here?"`

> 			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times, fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over, damn thing got me in more trouble than it was worth anyway."`

Reviewer 3: This guy is _old_, like, are Alpha's meant to have wildly extended lifespans?

Author: Elias is really really old, and I don't believe Betas are mentioned to be the only ones that got super long lifespans, so I assume Alphas also have them.

Reviewer 5: I can believe that.

> 			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times, fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over, damn thing got me in more trouble than it was worth anyway."`

Reviewer 5: Why would the Heliarchs demand his jump drive if they didn't demand it from the player?

Author: In telling them about the Alphas, Alitis would've explained how he's basically a living weapon, so if he intends to live among them then it's best he doesn't have a way to just bail out after committing some atrocity.

Reviewer 5: Yeah, that makes sense.

> 			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times, fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over, damn thing got me in more trouble than it was worth anyway."`

Reviewer 5: ```suggestion
			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times, fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over. Damn thing got me in more trouble than it was worth anyway."`
```

> 			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times, fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over. Damn thing got me in more trouble than it was worth anyway."`

Reviewer 4: ```suggestion
			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times - fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over. Damn thing got me in more trouble than it was worth anyway."`
```

			choice
				`	"So where are the other Alphas that came with you?"`
					goto others
				`	"What did the voice say?"`
			`	He forces a chuckle out. "You got here, so you have a jump drive, I assume? Go on, head east and maybe it'll speak to you too. Find out if it classifies you as 'disposable' as well."`
				goto end
			label others

> 			`	"Others? Oh, you misunderstood me. I took a crew of brigands and pirates, they're all long dead now, most of old age. I'd say they made the right choice in coming with me, no matter how much they protested that the aliens here were too alien."`

Reviewer 4: ```suggestion
			`	"Others? Oh, you misunderstood me. I took a crew of brigands and pirates. They're all long dead now, most of old age. I'd say they made the right choice in coming with me, no matter how much they protested that the aliens here were too... alien."`
```

			label end

> 			`	The Heliarchs inform you the medic has arrived, and have them treat your bruises. Although the Alpha was right in that you weren't injured much, the Heliarchs insist on cleaning up your bruises. Once they're done, an agent escorts you back to your ship.`

Reviewer 3: You must've bruised quick for this to happen.

Author: ?

Reviewer 5: https://www.thoughtco.com/what-is-a-bruise-4160373 <-- This website says that a red bruise can form right after an injury, so I could see this happening.

> 			`	The Heliarchs inform you the medic has arrived, and have them treat your bruises. Although the Alpha was right in that you weren't injured much, the Heliarchs insist on cleaning up your bruises. Once they're done, an agent escorts you back to your ship.`

Reviewer 5: ```suggestion
			`	The Heliarchs inform you that the medic has arrived, and have them treat your bruises. Although the Alpha was right in that you weren't injured much, the Heliarchs insist on cleaning up your injuries. Once they're done, an agent escorts you back to your ship.`
```

> 			`	The Heliarchs inform you that the medic has arrived, and have you go with them further down the hallway, to a room where they treat your bruises. Although the Alpha was right in that you weren't injured much, the Heliarchs insist on cleaning up your injuries. Once they're done, an agent escorts you back to your ship.`

Reviewer 4: ```suggestion
			`	The Heliarchs inform you that the medic has arrived, and bring you further down the hallway to a room where they treat your bruises. Although the Alpha was right in that you weren't injured much, the Heliarchs insist on cleaning up your injuries. Once they're done, an agent escorts you back to your ship.`
```

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

mission "Coalition: Alpha Encounter"
	minor
	invisible
	to offer
		has "Coalition: First Contact: done"
		has "remembrance day: offered"
		random < 1
	source
		government "Coalition"
		not attributes "station"
		not planet "Secret Sky" "Remote Blue" "Bloptab's Furnace" "Ahr"
	on offer
		log "Minor People" "Alitis" `Alitis, formerly known as Athis, is an Alpha who lives in the Coalition. Centuries ago, he stole a jump drive from an alien ship and used it to explore, but after some sort of encounter in the galactic east, he decided to give up on his former life.`
		conversation

> 			`You're browsing at a shop in the spaceport when you're suddenly tackled into the ground. You try to get up, but whoever attacked you pins you down, holding your arms behind your back to restrain you. When you manage to look back, you see a man with very long hair and an intricate necklace of gold. His most notable trait, however, is his green skin - an Alpha.`
> 			`	Your eyes don't meet for long, though, as he looks to his side to see an Arach that swoops in with a tackle. He lets go of your arms in an attempt to block the attack, but is too late. As you get up, they wrestle for a while, but it doesn't take long for the Arach Heliarch to restrain the Alpha. All eight legs firmly hold his arms to his torso as the Arach rests her entire body weight on the Alpha's back. A Saryd Heliarch approaches you, telling you to keep your distance. He then looks at the Alpha and asks, "What reason for attacking this person, did you have?"`
> 			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question and looks at you instead. He asks, "You're with the Navy, aren't you? Finally figured out I was here?"`

Reviewer 3: Suggestion added in fresh to accommodate for `outdated` source lines.

```suggestion
			`You're window-browsing a shop in the spaceport when you catch the scuff of two quick footsteps coming towards you. Before you can glance in that direction though you're suddenly tackled into the ground by someone astonishingly human-shaped. You try to get up, but whoever attacked you pins you down, pressing your face into the ground and holding your arms behind your back to restrain you. As you struggle against your attacker you manage to catch sight of long hair and then, with a flash of concern, a glimpse of bright green skin - an Alpha.`
			`	You are not pressed for long though, you feel their attention shift to the side you cannot see, as an Arach swoops in with a tackle. They lets go of your arms in an attempt to block the attack, but it does not prevent the Arach from bowling them clear off of you where the two begin to wrestle in your line of sight. The Alpha is a man and, surprisingly to you, despite his strength it doesn't take long for the Arach Heliarch to restrain your attacker. The Arach firmly pins his arms to his torso with four legs, holds his hips down with two, and uses her last pair to lever herself until she rests her entire body weight into pinning the Alpha.`
			`	A Saryd Heliarch approaches and tells you to keep your distance before interposing himself partially between you and your assailant. He then looks at the Alpha and asks through his translator box, "What reason for attacking this person, did you have?"`
			`	The Alpha is still struggling to try and get out of the Arach's grip, and you notice an intricate gold necklace which has splayed across his shoulder and onto the floor in the scuffle, jangling in the quiet every time he struggles as now everyone else has cleared out of the area and is giving the scene a wide berth. As you get to your feet he ignores the Saryd's question and looks at you intensely. "You're with the Navy, aren't you? Finally figured out I was here, thought you'd finish the job?"`
```

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

mission "Coalition: Alpha Encounter"
	minor
	invisible
	to offer
		has "Coalition: First Contact: done"
		has "remembrance day: offered"
		random < 1
	source
		government "Coalition"
		not attributes "station"
		not planet "Secret Sky" "Remote Blue" "Bloptab's Furnace" "Ahr"
	on offer
		log "Minor People" "Alitis" `Alitis, formerly known as Athis, is an Alpha who lives in the Coalition. Centuries ago, he stole a jump drive from an alien ship and used it to explore, but after some sort of encounter in the galactic east, he decided to give up on his former life.`
		conversation

> 			`You're window-browsing a shop in the spaceport when you catch the scuff of two quick footsteps coming towards you. Before you can glance in that direction, you're suddenly tackled into the ground by someone astonishingly human-shaped. You try to get up, but whoever attacked you pins you down, pressing your face into the ground and holding your arms behind your back to restrain you. As you struggle against your attacker you manage to catch sight of long hair and then, with a flash of concern, a glimpse of bright green skin - an Alpha.`

Reviewer 7: ```suggestion
			`You're window-browsing a shop in the spaceport when you catch the scuff of too quick footsteps coming towards you. Before you can glance in that direction, you're suddenly tackled into the ground by someone astonishingly human-shaped. You try to get up, but whoever attacked you pins you down, pressing your face into the ground and holding your arms behind your back to restrain you. As you struggle against your attacker you manage to catch sight of long hair and then, with a flash of concern, a glimpse of bright green skin - an Alpha.`
```

> 			`	You are not pressed for long though. You feel their attention shift to the side you cannot see, as an Arach swoops in with a tackle. They let go of your arms in an attempt to block the attack, but it does not prevent the Arach from bowling them clear off of you where the two begin to wrestle in your line of sight. The Alpha is a man and, surprisingly to you, despite his strength it doesn't take long for the Arach Heliarch to restrain your attacker. The Arach firmly pins his arms to his torso with four legs, holds his hips down with two, and uses her last pair to lever herself until she rests her entire body weight into pinning the Alpha.`

Reviewer 7: ```suggestion
			`	You are not pressed for long though. You feel their attention shift to the side you cannot see, as an Arach swoops in with a tackle. The Alpha lets go of your arms in an attempt to block the attack, but it does not prevent the Arach from bowling them clear off of you where the two begin to wrestle in your line of sight. The Alpha is a man and, surprisingly to you, despite his strength it doesn't take long for the Arach Heliarch to restrain your attacker. The Arach firmly pins his arms to his torso with four legs, holds his hips down with two, and uses her last pair to lever herself until she rests her entire body weight into pinning the Alpha.`
```

			`	A Saryd Heliarch approaches and tells you to keep your distance before interposing himself partially between you and your assailant. He then looks at the Alpha and asks through his translator box, "What reason for attacking this person, did you have?"`

> 			`	The Alpha is still struggling to try and get out of the Arach's grip, and you notice an intricate gold necklace which has splayed across his shoulder and onto the floor in the scuffle, jangling in the quiet every time he struggles as now everyone else has cleared out of the area and is giving the scene a wide berth. As you get to your feet he ignores the Saryd's question and looks at you intensely. "You're with the Navy, aren't you? Finally figured out I was here, thought you'd finish the job?"`

Reviewer 5: ```suggestion
			`	The Alpha is still struggling to try and get out of the Arach's grip, and you notice an intricate gold necklace which has splayed across his shoulder and onto the floor in the scuffle, jangling in the quiet every time he struggles. At this point, everyone else has cleared out of the area and is giving the scene a wide berth. As you get to your feet he ignores the Saryd's question and looks at you intensely. "You're with the Navy, aren't you? Finally figured out I was here, thought you'd finish the job?"`
```

			branch navy
				has "license: Navy"
			choice
				`	"No, I'm not with the Navy. You could've just asked without slamming me to the ground."`
					goto notnavy
				`	"Is that why you attacked me?"`
					goto detained
			label navy
			choice
				`	"I am with the Navy, but I didn't come here looking for you."`
				`	"Is that why you attacked me?"`
					goto detained

> 			`	"And I'm supposed to believe that crap?" He lashes out, trying to knock the Arach off of him, but the agent only tightens the grip. She opens her mandibles, getting them close to the Alpha's neck. "Alright, alright, I get it!" the Alpha shouts, stopping his attempts to break free.`

Reviewer 7: ```suggestion
			`	"And I'm supposed to believe that crap?" He lashes out, trying again to knock the Arach off of him, but the agent only tightens the grip. She opens her mandibles, getting them close to the Alpha's neck. "Alright, alright, I get it!" the Alpha shouts, stopping his attempts to break free.`
```

				goto detained
			label notnavy
			`	He stares at you for a moment, as if unsure if you're telling the truth. "What the hell are you doing here then? Did you just stumble onto this place?" He stops struggling, but the Arach agent doesn't loosen her grip.`
			label detained

> 			`	The Saryd walks closer to the Alpha, stomping his hoofs on the ground with each step. "Whatever your motivation was, a transgression, you have committed. Punished accordingly, you shall be." He says. "Kindly follow us to the detention facility, would you, Captain?" He asks you, but doesn't wait for an answer, as more agents arrive to escort you and the Alpha to the "detention facility."`

Reviewer 7: ```suggestion
			`	The Saryd walks closer to the Alpha, stomping his hoofs on the ground with each step. "Whatever your motivation was, a transgression, you have committed. Punished accordingly, you shall be," he says. "Kindly follow us to the detention facility, would you, Captain?" he asks you, but doesn't wait for an answer, as more agents arrive to escort you and the Alpha to the "detention facility."`
```

			`	It's a short walk, and you end up in a building similar to a police station. The Alpha is brought before a Kimek that you guess is in charge of the facility. He asks you to give a testimony of what happened, with the Heliarch agents confirming what you say. "Your first offense in a long time this is, Alitis," the Kimek says, after checking a computer for the Alpha's data. "Three years in seclusion, your sentence is."`

> 			`	After the short verdict, which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building, with several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though, all of the cells appear to be empty, and they simply lock him in the second one where accessing his cell would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive so that they can take care of your bruises.`

Reviewer 5: ```suggestion
			`	After the short verdict, which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building. This floor has several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though, all of the cells appear to be empty, and they simply lock him in the second one where accessing his cell would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive so that they can take care of your bruises.`
```

> 			`	After the short verdict, which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building. This floor has several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though, all of the cells appear to be empty, and they simply lock him in the second one where accessing his cell would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive so that they can take care of your bruises.`

Reviewer 7: ```suggestion
			`	After the short verdict, which seems to lack much of the due process you would expect in human space, the agents bring both you and the Alpha to a lower level of the building. This floor has several empty cells kept shut by thick, tough walls of some transparent material. While the cell block does appear to go further beyond this section and around a corner, the first few cells appear to be directly opposite a modest medical space, presumably used mostly for treating any injured prisoners without leaving the secure level. At present though, all of the cells appear to be empty, and the Heliarchs simply lock him in the second cell, where accessing his would not obstruct any other doorways. The agents then ask that you wait a few minutes for a medic to arrive so that they can take care of your bruises.`
```

			choice
				`	"You 'knew better?' What do you mean?"`
				`	"What, did you have a change of heart once you got here?"`
			`	He sits on the ground for a while, thinking. "I was one of those that fought humanity after the first Turf War. I was ambushed, cornered many times - fear wasn't a stranger to me. After we stole that jump drive and ventured east, however, someone or... something rather, some voice spoke to me. It was the first time I felt scared. After that, I wasn't inclined to go back east with the others, no matter how silly a reason they thought it was. When I got here, me and my crew helped the Heliarchs with translations for humans, and I told them about humanity and the Alphas. They took me in, but demanded my jump drive, so I handed it over. Damn thing got me in more trouble than it was worth anyway."`
			choice

> 				`	"So where are the other Alphas that came with you?"`
> 					goto others
> 				`	"What did the voice say?"`
> 			`	He forces a chuckle out. "You got here, so you have a jump drive, I assume? Go on, head east and maybe it'll speak to you too. Find out if it classifies you as 'disposable' as well."`
> 				goto end
> 			label others
> 			`	"Others? Oh, you misunderstood me. I took a crew of brigands and pirates. They're all long dead now, most of old age. I'd say they made the right choice in coming with me, no matter how much they protested that the aliens here were too... alien."`
> 			label end

Reviewer: Changing the choice since I removed the mention of other crew, and also expanding upon the responses a bit.
```suggestion
				`	"Voices in your head? Are you sure you weren't just imagining them?"`
					goto imaginary
				`	"What did the voice say?"`
			`	He looks away from you. "You're here now, aren't you? This means you must have a device to jump beyond links as I did. Travel to the east and find out for yourself what the voices say. Find out if it classifies you as 'disposable' as well.`
				goto end
			label imaginary
			`	He looks away from you. "No imaginary thing could have such an impact on me. You're here now, aren't you? This means you must have a device to jump beyond links as I did. Travel to the east and find out for yourself that the voices are real."`
			label end
```

> 			`	The Heliarchs inform you that the medic has arrived, and bring you further down the hallway to a room where they treat your bruises. Although the Alpha was right in that you weren't injured much, the Heliarchs insist on cleaning up your injuries. Once they're done, an agent escorts you back to your ship.`

Reviewer: When did he say anything about my injuries?
Perhaps change this up to mention how Alitis looks as you leave. Since you're brought further down the hallway, you probably come back up the hall after you've been patched up. I suggested having Alitis turn away from you after your last question, so perhaps you walk past him still staring at the wall.

					goto notnavy
				`	"Is that why you attacked me?"`
					goto detained
			label navy
			choice
				`	"I am with the Navy, but I didn't come here looking for you."`
				`	"Is that why you attacked me?"`
					goto detained
			`	"And I'm supposed to believe that crap?" He lashes out, trying again to knock the Arach off of him, but the agent only tightens the grip. She opens her mandibles, getting them close to the Alpha's neck. "Alright, alright, I get it!" the Alpha shouts, stopping his attempts to break free.`
				goto detained
			label notnavy
			`	He stares at you for a moment, as if unsure if you're telling the truth. "What the hell are you doing here then? Did you just stumble onto this place?" He stops struggling, but the Arach agent doesn't loosen her grip.`
			label detained
			`	The Saryd walks closer to the Alpha, stomping his hoofs on the ground with each step. "Whatever your motivation was, a transgression, you have committed. Punished accordingly, you shall be," he says. "Kindly follow us to the detention facility, would you, Captain?" he asks you, but doesn't wait for an answer, as more agents arrive to escort you and the Alpha to the "detention facility."`

> 			`	On your way there, you start feeling stings on your arms and legs, and realize the Alpha's tackle left you with some bruises. "Treat those, we will, after faced trial, your agressor has." The Saryd says as you approach your destination, a building similar to a police station. The Alpha is brought before a Kimek that you guess is in charge of the facility. He asks you to give a testimony of what happened, with the Heliarch agents confirming what you say. "Your first offense in a long time this is, Alitis," the Kimek says, after checking a computer for the Alpha's data. "Three years in seclusion, your sentence is."`

Reviewer: ```suggestion
			`	On your way there, you start feeling stings on your arms and legs, and realize the Alpha's tackle left you with some bruises. "Treat those, we will, after faced trial, your agressor has," the Saryd says as you approach your destination: a building similar to a police station. The Alpha is brought before a Kimek that you guess is in charge of the facility. He asks you to give a testimony of what happened, with the Heliarch agents confirming what you say. "Your first offense in a long time this is, Alitis," the Kimek says, after checking a computer for the Alpha's data. "Three years in seclusion, your sentence is."`
```

			`	You notice that cut into the transparent material of the walls are a series of small holes at head-height with a cover that can be slid across them, presumably to silence a prisoner if needed. By default, they appear to be left open, and the Heliarch agents have not closed the one on the Alpha's cell.`
			`	Your eyes shift from the holes in the wall to the Alpha to find his eyes fixated on you, not looking away or even blinking. After you stare him down for a couple of seconds, he speaks up. "What is it, are you expecting some sort of apology? I'd consider my caution to be entirely justified, so you won't get one."`
			choice
				`	"Who are you? How did you even get here?"`
				`	"What are you doing here in the Coalition?"`
			`	He frowns at you. "I don't owe you any answers, cur. Shut up and wait for the law to bring the frail princess some band-aids and a lollipop."`
			choice
				`	"Are you so busy in that empty cell that you can't answer me?"`
				`	"Tough talk for someone who wrestled a spider and lost."`

> 			`	He leans forward, his legs twitching as if stopped mid-tackle, and he pauses, turning to look down the hallway to where the Heliarchs went. He backs up, slowing his breathing, and then sits down. "Alright. Might be the last time in three years I can have a proper conversation, so might as well entertain you. My name... is Alitis. Over a century ago my kin stumbled upon an alien civilization beyond the Far North in conflict with bands of invaders. These invaders possessed drive capable of jumping between systems without the need for hyperspace links, technology that we greatly desired, for it could put us far outside of the reach of... you." He fixates his eyes on yours, ever so slightly closing his eyelids. "We explored the far reaches of the galaxy to the east, but found no place to call home. Our only choice was to return to the Dirt Belt where we knew of an enclave of more kin, only for the Navy," he spits on the ground, then looks at you again, "to find us. We evacuated, and my kin thought it wise to return to the east, but I knew better. I stole one of the drives my kin had along with a small ship and began traveling west. That is when I discovered the Coalition, and they have provided for me a home that I have been unable to find elsewhere in the galaxy."`

Reviewer: Slight reword on one sentence.
`Might be the last` -> `This may be the last`
`might as well entertain you` -> `I will entertain you`
```suggestion
			`	He leans forward, his legs twitching as if stopped mid-tackle, and he pauses, turning to look down the hallway to where the Heliarchs went. He backs up, slowing his breathing, and then sits down. "Alright. This may be the last time in three years I can have a proper conversation, so I will entertain you. My name... is Alitis. Over a century ago my kin stumbled upon an alien civilization beyond the Far North in conflict with bands of invaders. These invaders possessed drive capable of jumping between systems without the need for hyperspace links, technology that we greatly desired, for it could put us far outside of the reach of... you." He fixates his eyes on yours, ever so slightly closing his eyelids. "We explored the far reaches of the galaxy to the east, but found no place to call home. Our only choice was to return to the Dirt Belt where we knew of an enclave of more kin, only for the Navy," he spits on the ground, then looks at you again, "to find us. We evacuated, and my kin thought it wise to return to the east, but I knew better. I stole one of the drives my kin had along with a small ship and began traveling west. That is when I discovered the Coalition, and they have provided for me a home that I have been unable to find elsewhere in the galaxy."`
```
